R/unlikely_survey_area.R

```diff
@@ -0,0 +1,58 @@
+#' Detect surveys with an unlikely surveyed area
+#'
+#' First we create a Delaunay triangulation of the survey area.
+#' Then we ignore all triangles with an edge larger than `max_edge`.
+#' Finally we calculate the area of the remaining triangles.
+#' When the sum of the area of the remaining triangles is larger than
+#' `max_area`, we consider the survey area unlikely.
+#' @inheritParams import_observations
+#' @inheritParams sf::st_as_sf
+#' @inheritParams sf::st_buffer
+#' @inheritParams sf::st_concave_hull
+#' @param relevant_area An `sf` object with the relevant area.
```

Comment:
#' @param dist The size of the buffer (in meters) around each observation.
#' Default of 50.
#' @param crs The coordinate reference system for the observations and the relevant area.
#' Default of 31370 (Belgian Lambert 72).
#' @param ratio A numeric value which controls the concavity of the resulting hull around the observations
#' Default of 0.95